template/v1/dirs/etc/supervisor/conf.d/supervisord-jupyter-lab.conf

```diff
@@ -0,0 +1,11 @@
+[include]
```

Comment:
this file is not used as of now - I've just refactored in this PR to create `supervisord-common.conf`
Will raise a subsequent PR to make that change